ui/lib/core/addon/components/form-field.js

```diff
@@ -75,8 +75,18 @@ export default class FormFieldComponent extends Component {
     this.showInput = !!modelValue;
   }
 
+  get columnRadio() {
```

Comment:
We want this to be true if _any_ possible option has help/sub text right? I think update to something like the following might be a little more durable. There may be a case where one option doesn't have subtext, but another does

```js
this.args?.attr?.options?.possibleValues?.any((v) => v.helperText)